packages/metal-soy/test/Soy.js

```diff
@@ -11,6 +11,7 @@ import { Events as EventsComponent } from './assets/Events.soy.js';
 // on it.
 // TODO: We should have a better dependency management for soy files so that
 // the order in which they're required doesn't matter.
+import { ComputedData as ComputedDataComponent } from './assets/ComputedData.soy.js';
```

Comment:
Could you make sure this is the right line to place the `import`? I simply sorted them alphabetically.